django_cloud_tasks/tasks/task.py

```diff
@@ -69,6 +85,9 @@ def from_headers(cls, headers: dict) -> Self:
             eta=eta,
             previous_response=headers.get(f"{cloud_tasks_prefix}TaskPreviousResponse"),
             previous_failure=headers.get(f"{cloud_tasks_prefix}TaskRetryReason"),
+            is_cloud_scheduler=is_cloud_scheduler,
+            cloud_scheduler_schedule_time=schedule_time,
+            cloud_scheduler_job_name=cloud_scheduler_job_name,
         )
 
     def to_headers(self) -> dict:
```

Comment:
TODO maybe update `to_headers` too? It is only used by unit tests, and because I particularly didn't run into tests that would need this implementation to be augmented, I didn't touch it :-P

Reply:
Yes, please, it's a not brainer to add support for these optional headers.
This method is really useful when running/testing locally, it's great to create a scenario that is very close to the cloud.